### PR TITLE
Azure Stack KeyVault Admin SDK

### PR DESCRIPTION
**Pull request by deathly809**
## Description

Azure Stack requires an admin interface for KeyVault.  Currently we only support infinite quotas ha ha.

https://github.com/Azure/azure-rest-api-specs/pull/2779

---

This checklist is used to make sure that common guidelines for a pull request are followed.
- [X] Please add REST spec PR link to the SDK PR
- [X] **I have read the [contribution guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md).**
- [X] **The pull request does not introduce [breaking changes](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/Documentation/breaking-changes.md).**

### [General Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md#general-guidelines)
- [X] Title of the pull request is clear and informative.
- [X] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For more information on cleaning up the commits in your PR, [see this page](https://github.com/Azure/azure-powershell/blob/dev/documentation/cleaning-up-commits.md).

### [Testing Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md#testing-guidelines)
- [X] Pull request includes test coverage for the included changes.

### [SDK Generation Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md#sdk-generation-guidelines)
- [X] If an SDK is being regenerated based on a new swagger spec, a link to the pull request containing these swagger spec changes has been included above.
- [X] The generate.cmd file for the SDK has been updated with the version of AutoRest, as well as the commitid of your swagger spec or link to the swagger spec, used to generate the code.
- [X] The `*.csproj` and `AssemblyInfo.cs` files have been updated with the new version of the SDK.

### CODE REVIEW

File: src/AzureStack/KeyVaultAdmin/KeyVault.Admin/Microsoft.AzureStack.Management.KeyVault.Admin.csproj [hunk 1]
```diff
@@ -0,0 +1,15 @@
+﻿
+<Project Sdk="Microsoft.NET.Sdk">
+  <Import Project="$([MSBuild]::GetPathOfFileAbove('AzSdk.reference.props'))" />
+  <PropertyGroup>
+    <PackageId>Microsoft.AzureStack.Management.KeyVault.Admin</PackageId>
+    <Description>Provides developers with libraries for the KeyVault admin under AzureStack.</Description>
+    <VersionPrefix>0.1.0-preview</VersionPrefix>
```
Comment (dsgouda) [inline src/AzureStack/KeyVaultAdmin/KeyVault.Admin/Microsoft.AzureStack.Management.KeyVault.Admin.csproj]: We recommend starting preview versioning from `0.9.0-preview`

File: src/AzureStack/KeyVaultAdmin/KeyVault.Admin/generate.cmd [hunk 2]
```diff
@@ -0,0 +1,7 @@
+::
+:: Microsoft Azure SDK for Net - Generate library code
+:: Copyright (C) Microsoft Corporation. All Rights Reserved.
+::
+
+@echo off
+call %~dp0..\..\..\..\tools\generate.cmd azsadmin/resource-manager/keyvault latest deathly809 azs.keyvault.admin azure-rest-api-specs %CD%
```
Comment (dsgouda) [inline src/AzureStack/KeyVaultAdmin/KeyVault.Admin/generate.cmd]: Code should not be generated from the `deathly809` fork

**Review by dsgouda — CHANGES_REQUESTED** [review]
Please generate the code using the generate.cmd or generate.ps1 scripts here. This should generate a SdkInfo_*.cs file and a .txt file, please commit these files

File: src/AzureStack/KeyVaultAdmin/KeyVault.Admin/Microsoft.AzureStack.Management.KeyVault.Admin.csproj (hunk 1 above)
Comment (deathly809) [reply src/AzureStack/KeyVaultAdmin/KeyVault.Admin/Microsoft.AzureStack.Management.KeyVault.Admin.csproj]: We already have modules which were published starting at 0.1.0.

**Comment by dsgouda** [comment]
@azuresdkci retest this please

**Comment by dsgouda** [comment]
@azuresdkci retest

**Comment by deathly809** [comment]
@dsgouda Regenerated/updated.

File: src/AzureStack/KeyVaultAdmin/AzSdk.RP.props [hunk 3]
```diff
@@ -0,0 +1,7 @@
+<Project ToolsVersion="15.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <!--This file and it's contents are updated at build time moving or editing might result in build failure. Take due deligence while editing this file-->
+  <PropertyGroup>
+    <AzureApiTag />
```
Comment (dsgouda) [inline src/AzureStack/KeyVaultAdmin/AzSdk.RP.props]: Please run `msbuild build.proj /t:build /p:Scope=src\AzureStack\KeyVaultAdmin\` and commit the changes that should be on this file.

**Review by dsgouda — CHANGES_REQUESTED** [review]
Looks good for the most part, one minor comment.

**Pull request merged**